### PR TITLE
Task 5

### PR DESCRIPTION
**Pull request by yshurhay**

### CODE REVIEW

File: UI/main.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```
Comment (smiakchilo) [inline UI/main.html:1]: These files should not be present in the current PR. They belong to other tasks

File: task5/src/Servlet/WriteName.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class WriteName extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String s = req.getParameter("name");
```
Comment (smiakchilo) [inline task5/src/Servlet/WriteName.java:12]: A check for name length is missing

**Review by smiakchilo — APPROVED** [review]
Please observe the scope of each PR. Files not belonging to a task should never be includes in a PR (in commercial development as well - and for the major part there)
Also, remember to use String constants insted of "magic literals" when dealing withString values. Like `someWriter.print(JSON_SUCCESS_VALUE)` instead of `someWriter.print("{\"success\": true}")` with all these ugly escaptings. This is a good habit)

**Comment by MaksimViarenich** [comment]
Merge this PR, please

**Pull request merged**